dm/ctl/simulator/event_filter.go

```diff
@@ -0,0 +1,115 @@
+// Copyright 2019 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package simulator
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/pingcap/dm/dm/config"
+	"github.com/pingcap/dm/dm/ctl/common"
+	"github.com/pingcap/dm/dm/pb"
+	"github.com/pingcap/errors"
+	_ "github.com/pingcap/tidb/types/parser_driver" // for import parser driver
+
+	"github.com/spf13/cobra"
+)
+
+type eventFilterResult struct {
+	WillBeFiltered string `json:"will-be-filtered"`
+	FilterName     string `json:"filter-name,omitempty"`
```

Comment:
1. Can we unify the name `filter-name` with `reason` in `table-route`?
2. Is it possible to show the reason for `bw-list`?